src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -245,8 +245,8 @@ private void OnPointerPressed(object sender, PointerRoutedEventArgs args)
 		)
 		{
 			// The check is here because ContentDialogPopupPanel returns true for IsViewHit() even though light-dismiss is always
-			// disabled for ContentDialogs
-			if (popup.IsLightDismissEnabled)
+			// disabled for ContentDialogs. In addition, CommandBar is always closes the Popup.
```

Comment:
Probably the popup should have had IsLightDismissEnabled in the first place

Reply:
Nope. That's the problem. It dismisses regardless of IsLightDismissEnabled.

Reply:
Interesting. Could you try setting `IsSticky` on Windows and see if it changes the behavior?

Reply:
I think your change should have failed `ValidateInlineCommandBarLightDismissBehavior`:

https://github.com/unoplatform/uno/blob/e5b91a50ed1e3d469b24f4e67caad1b0c7c8741f/src/Uno.UI.RuntimeTests/IntegrationTests/dxaml/controls/commandbar/CommandBarIntegrationTests.cs#L988

I don't see why this test didn't fail, but that means we have a problem that the test doesn't validate the behavior properly (somehow).

Reply:
The dismiss should happen through `TryDismissInlineAppBar` (likely in `OnOverlayElementPointerPressed`).

See https://github.com/unoplatform/uno/blob/e5b91a50ed1e3d469b24f4e67caad1b0c7c8741f/src/Uno.UI/UI/Xaml/Controls/AppBar/AppBar.Partial.cs#L1305


and

https://github.com/unoplatform/uno/blob/e5b91a50ed1e3d469b24f4e67caad1b0c7c8741f/src/Uno.UI/UI/Xaml/Controls/AppBar/AppBar.Partial.cs#L1281-L1291